seokhwan/2/두_정수사이의_합.js

```diff
@@ -0,0 +1,12 @@
+function solution(a, b) {
+  if (a > b) {
+    const temp = a;
+    a = b;
+    b = temp;
+  }
+
+  let sum = a;
+
+  for (++a; a <= b; a++) sum += a;
```

Comment:
```suggestion
  for (++a; a <= b; a++) {
  sum += a;
  };
```
중괄호로 묶어줄 수 있도록 합시다~~